### ISSUE TITLE
Missing Parsers problem?

### ISSUE THREAD

**Issue by natemcintosh** [JuliaData/CSV.jl#308]
when using CSV 0.4 I always get this message when `using CSV` . 

`(julia> using CSV
[ Info: Recompiling stale cache file /Users/mcintna1/.julia/compiled/v1.0/CSV/HHBkp.ji for CSV [336ed68f-0bac-5ca0-87d4-7b16caf5d00b]
┌ Warning: Package CSV does not have Parsers in its dependencies:
│ - If you have CSV checked out for development and have
│   added Parsers as a dependency but haven't updated your primary
│   environment's manifest file, try Pkg.resolve().
│ - Otherwise you may need to report an issue with CSV
└ Loading Parsers into CSV from project dependency, future warnings for CSV are suppressed.`

At first I thought I might just need to add `Parsers`, but it already seems to be there.
`(v1.0) pkg> add Parsers`

  `Updating registry at ~/.julia/registries/General
  Updating git-repo https://github.com/JuliaRegistries/General.git
 Resolving package versions...
  Updating ~/.julia/environments/v1.0/Project.toml
 [no changes]
  Updating ~/.julia/environments/v1.0/Manifest.toml
 [no changes])`

**Comment by kklot** [comment]
Just adding to the above post, fresh started Julia has the same issue

```
$ exec '/Applications/Julia-1.0.app/Contents/Resources/julia/bin/julia'
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.0.0 (2018-08-08)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> using CSV
[ Info: Precompiling CSV [336ed68f-0bac-5ca0-87d4-7b16caf5d00b]
ERROR: LoadError: ArgumentError: Package CSV does not have Parsers in its dependencies:
- If you have CSV checked out for development and have
  added Parsers as a dependency but haven't updated your primary
  environment's manifest file, try `Pkg.resolve()`.
- Otherwise you may need to report an issue with CSV
``` 

I ran `resolve` and the issue persists

```
(v1.0) pkg> resolve
 Resolving package versions...
  Updating `~/.julia/environments/v1.0/Project.toml`
 [no changes]
  Updating `~/.julia/environments/v1.0/Manifest.toml`
 [no changes]
```

**Comment by laborg** [comment]
It appears that this has been fixed in CSV 4.1.0 but this still needs to get into the general registry.

**Comment by quinnj** [comment]
Yes, the tag has been merged: https://github.com/JuliaLang/METADATA.jl/pull/18413. Once it syncs to the general registry, we should be good.
